.rubocop.yml

```diff
@@ -69,6 +69,8 @@ RSpec/ExpectInHook:
   Enabled: false
 RSpec/ExampleLength:
   Max: 25
+RSpec/Focus:
+  AutoCorrect: false
```

Comment:
Je me demandais, est-ce qu'on voulait aussi l'équivalent pour `Lint/Debugger` comme mentionné dans le post linké?

Reply:
No, because the `Lint/Debugger` does not support autocorrection.